std_msgs/Int32 view
---
string result